test/cache_test.rb

```diff
@@ -146,6 +151,49 @@ class InheritedRoleSerializer < RoleSerializer
       @blog_serializer     = BlogSerializer.new(@blog)
     end
 
+    def test_expiring_of_cache_at_update_of_record
+      if ARModels::Author.respond_to?(:cache_versioning)
+        ARModels::Author.cache_versioning = true
+      end
+
+      author = ARModels::Author.create(name: 'Foo')
+      author_json = AuthorSerializerWithCache.new(author).as_json
+
+      assert_equal 'Foo', author_json[:name]
+
+      author.update_attributes(name: 'Bar')
```

Comment:
```suggestion
      author.update!(name: 'Bar')
```

Reply:
Trying to understand why the change to the first `update!` is failing on Travis first

Reply:
I have missed this comment sorry or I would have made the change myself :(